spec/support/jsonapi_schema.json

```diff
@@ -0,0 +1,397 @@
+{
```

Comment:
Probably there should be a spec or CI task that checks if this is most recent version of schema

Reply:
This is a super interesting issue and I actually added it in the original PR message. I don't know how to tackle this properly. Having it as a spec within the JsonApi::Parameters seems interesting, but then it may create an issue when you're working on something completely unrelated to JSON schema validation.

For instance, if you work on, lets say, some parsing logic changes, you changed small portion of code in the default handlers, you run `bin/ci` or any other RSpec runner and you get an error saying the JSON Schema needs an update - I wouldn't want this to happen, as it will lure the developer away from what (s)he was doing. :thinking: 

I would love to hear your thoughts on this.

Reply:
It seems that JSONSchemer [has some external reference dialup](https://github.com/davishmcclurg/json_schemer#options) so potentially maybe there's a way for it to fetch the schema at least in the specs? 

We have to also keep in mind that whatever is delivered in user applications _has to have_ a cached / predownloaded version of the schema so we wouldn't trigger a request in user's Rails application whenever he gets a `TranslatorError`.

Reply:
Then probably separate CI task is a best way out of this situation. One that can fail (is optional)

Reply:
From experience I can say, that some web application servers do not have access to internet as security precaution

Reply:
I agree with you guys. On top of that, we not only have to have to most up to date version of schema but we also need to make sure that the gem handles it properly. So it may not be so automatic after all. 
Definitely something to think about. I would just not freeze this change until we find the best solution. Let's plan it for 2.0 and move forward.

Reply:
That sounds like a good plan. I'm adding this as a task for later.